RF24.h

```diff
@@ -1559,8 +1589,8 @@ class RF24 {
      * Request (IRQ) pin active LOW.
      * The following events can be configured:
      * 1. "data sent": This does not mean that the data transmitted was
-     * received, only that the attempt to send it was complete.
-     * 2. "data failed": This means the data being sent was not received. This
+     * recieved, only that the attempt to send it was complete.
+     * 2. "data failed": This means the data being sent was not recieved. This
```

Comment:
The spelling for the word "received" has already been corrected in #817. Please rebase your master branch on upstream master.

Reply:
I'm still trying to figure out how to rebase from the upstream master.

Reply:
I checked your branch's commit history, and it looks like the merge commit from #817 is in your branches history already. So, I don't know why this is showing up in the diff. Although, it should be easy to replace all "recieved" with the properly spelled "received".

Reply:
I meant that you should re-do the search-and-replace all... Please respect the work that's been done and don't add to it (especially if its repetitive).

Reply:
I see, I have been using the client now hopefully correctly.  Thanks for taking the time to explain this stuff to me, I appreciate it.